src/main/resources/lib/ssb/utils/dateUtils.ts

```diff
@@ -42,14 +44,32 @@ export function formatDate(date: string | undefined, formatType: string, languag
         }
       : {}
 
+    let dateFnsResult
+
     try {
-      const result = format(parsedDate, formatType, locale)
-      return result
+      dateFnsResult = format(parsedDate, formatType, locale)
     } catch (e) {
       log.error(`Error in formatDate, tried to format ${parsedDate} to ${formatType}`)
-      log.error(JSON.stringify(locale, null, 2))
-      throw e
     }
+
+    let libTimePattern = formatType
+    // https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/format/DateTimeFormatter.html#patterns
+    if (formatType === 'PPP') libTimePattern = 'd. MMMM u'
```

Comment:
The expected pattern for the english formatted date is without the . for date e.g. 25 September 2023.
The error logs are firing off for that reason.